.jenkins/pytorch/win-test-helpers/test_python_nn.bat

```diff
@@ -0,0 +1,22 @@
+call %TMP_DIR%/ci_scripts/setup_pytorch_env.bat
+:: Some smoke tests
+cd test
+:: Checking that caffe2.python is available
```

Comment:
Better to change these comments as echo clauses, otherwise the users may be confused.